### PR TITLE
Storing and updating join split nullifers

### PR DESCRIPTION
**Pull request by NikVolf**

### CODE REVIEW

File: db/src/kv/transaction.rs [hunk 1]
```diff
@@ -54,6 +58,8 @@ pub enum Value {
 	TransactionMeta(TransactionMeta),
 	BlockNumber(u32),
 	Configuration(Bytes),
+	Bloom(Bytes),
```
Comment (svyatonik) [inline db/src/kv/transaction.rs]: Is this (`Bloom`, `Empty`) a part of some future PR or an redundant artifact?

Comment (svyatonik) [reply db/src/kv/transaction.rs]: Sorry - `Empty` is used, but `Bloom` is not

Comment (NikVolf) [reply db/src/kv/transaction.rs]: Ah, right

I was going to add bloom as well, but decided to add later

Comment (NikVolf) [reply db/src/kv/transaction.rs]: With those reorgs it is unclear whether bloom is going to be useful

**Pull request merged**